src/templates/core/proposals/assign/start_proposal.html

```diff
@@ -39,25 +39,30 @@
         {% bootstrap_form default_fields %}
         {% if proposal_form %}
           <div class="row">
-         {% for field in proposal_form %}
-            <div class="{% if field.field.widget.attrs.div_class %}{{ field.field.widget.attrs.div_class }}{% else %}col-md-12{% endif %}">
+          {% for field in proposal_form %}
+            <div class="{% if field.field.widget.attrs.div_class %}{{ field.field.widget.attrs.div_class }}
+            {% else %}col-md-12{% endif %}">
```

Comment:
These could be simplified to `{{ field.field.widget.attrs.div_class|default:"col-md-12" }}`.